guides/v2.2/cloud/project/project-conf-files_services-rabbit.md

```diff
@@ -108,6 +108,7 @@ For debugging purposes, it is useful to directly connect to a service instance i
    ```
 
 1. While the session is open, you can start a RabbitMQ client of your choice from your local workstation, configured to connect to the `localhost:<portnumber>` using the port number, username, and password information from the MAGENTO_CLOUD_RELATIONSHIP variable.
+	 The RabbitMQ management plugin is installed in cloud, so there is a graphical utility that can be used for debugging purposes and it is usually available at port 15672.
```

Comment:
I'm running into some difficulty finding anyone who uses this. In any case, I would make the following changes:

```suggestion

   To further assist in debugging, you can use the management plugin to access a graphical utility by opening the `http://localhost:15672` URL in your browser.
```